lib/plugins/aws/rollbackFunction/index.js

```diff
@@ -0,0 +1,123 @@
+'use strict';
+
+const BbPromise = require('bluebird');
+const validate = require('../lib/validate');
+const fetch = require('node-fetch');
+
+class AwsRollbackFunction {
+  constructor(serverless, options) {
+    this.serverless = serverless;
+    this.options = options || {};
+    this.provider = this.serverless.getProvider('aws');
+
+    Object.assign(this, validate);
+
+    this.commands = {
+      rollback: {
+        commands: {
+          function: {
+            usage: 'Rollback the function to a specific version',
+            lifecycleEvents: [
+              'rollback',
+            ],
+            options: {
+              function: {
+                usage: 'Name of the function',
+                shortcut: 'f',
+                required: true,
+              },
+              version: {
+                usage: 'Version of the function',
+                shortcut: 'v',
+                required: true,
+              },
+              stage: {
+                usage: 'Stage of the function',
+                shortcut: 's',
+              },
+              region: {
+                usage: 'Region of the function',
+                shortcut: 'r',
+              },
+            },
+          },
+        },
+      },
+    };
+
+    this.hooks = {
+      'rollback:function:rollback': () => BbPromise.bind(this)
+        .then(this.validate)
+        .then(this.getFunctionToBeRestored)
+        .then(this.fetchFunctionCode)
+        .then(this.restoreFunction),
+    };
+  }
+
+  getFunctionToBeRestored() {
+    const funcName = this.options.function;
+    let funcVersion = this.options.version;
+
+    // versions need to be string so that AWS understands it
+    funcVersion = String(this.options.version);
```

Comment:
Good point. I've added a validation for that below https://github.com/serverless/serverless/blob/512862fce00a80583cd92d73db8d0ea0630acdfc/lib/plugins/aws/rollbackFunction/index.js#L80-L91

The problem is that you could also provide `$LATEST` so it's not always a number.